GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -31,30 +48,70 @@ protected override string VerbName
 
         public override void Execute()
         {
-            this.localConfig = new LocalGVFSConfig();
+            if (GVFSPlatform.Instance.IsUnderConstruction)

```

Comment:
The immediate need is to decide the location where we want to store our config file. The equivalent of "C:\ProgramData\GVFS" on Mac.

Reply:
There's also (currently) no point in enabling `gvfs config` until we enable `gvfs upgrade`

Reply:
Ok thanks!

> The immediate need is to decide the location where we want to store our config file. The equivalent of "C:\ProgramData\GVFS" on Mac

This is a great question (and outside the scope of this PR).  I have not spent anytime looking into where applications typically store machine wide configurations on macOS.

Reply:
>>I have not spent anytime looking into where applications typically store machine wide configurations on macOS.

Native Mac app settings are stored in a hierarchy, system wide settings go in ```/Library/Preferences``` and user level settings in ```~/Library/Preferences```. Since we manage our own config settings we can decide to keep it in a place that makes the most sense to us (may be in ```/usr/local/GVFS```.